docs/functions/functions.json

```diff
@@ -437,6 +437,77 @@
 				true
 			]
 		},
+		{
+			"name": "max",
+			"id": "max",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Calculate the maximum of all numbers in the list.",
+			"argument": [
+				{
+					"name": "list",
+					"optional": false,
+					"type": "List of numbers",
+					"description": "The numbers you are trying to find the max of."
+				}
+			],
+			"returnType": "Number",
+			"returnDescription": "The minimum number"
+		},
+		{
+			"name": "round",
+			"id": "round",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Round a given number to the specified decimal place. Round to whole number if decimal place not specified.",
+			"argument": [
+				{
+					"name": "number",
+					"optional": false,
+					"type": "Number",
+					"description": "Number to be rounded."
+				},
+				{
+					"name": "decimals",
+					"optional": false,
+					"type": "Number",
+					"description": "Number of decimal places to the round the number to."
+				}
+			],
+			"returnType": "Number",
+			"returnDescription": "The rounded number."
+		},
+		{
+			"name": "digits",
+			"id": "digits",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Returns a number with the specified number of digits and decimal places.",
+			"argument": [
+				{
+					"name": "digits",
+					"optional": true,
+					"type": "Number",
+					"description": "Number of digits to keep."
+				},
+				{
+					"name": "decimals",
+					"optional": true,
+					"type": "Number",
+					"description": "Number of decimal places to keep."
```

Comment:
Give a little more detail. What happens if the number has less or more than these? Same for the one above.